src/test/kotlin/com/sourcegraph/cody/agent/protocol/PositionTest.kt

```diff
@@ -6,58 +6,78 @@ class PositionTest : BasePlatformTestCase() {
 
   private val content = "Hello\nWorld"
   private val filename = "position_test.txt"
-  private val file by lazy { myFixture.createFile(filename, content) }
-  private val document by lazy { myFixture.editor.document }
 
-  override fun setUp() {
-    super.setUp()
+  fun test_isStartOrEndOfDocumentMarkerReturnsTrueWhenLineIsLessThanZero() {
+    val file by lazy { myFixture.createFile(filename, content) }
+    val document by lazy { myFixture.editor.document }
     myFixture.openFileInEditor(file)
-  }
 
-  fun test_isStartOrEndOfDocumentMarkerReturnsTrueWhenLineIsLessThanZero() {
     val position = Position(-1, 0)
     val result = position.isStartOrEndOfDocumentMarker(document)
     assertEquals(true, result)
   }
 
   fun test_isStartOrEndOfDocumentMarkerReturnsTrueWhenLineIsGreaterThanLineCount() {
```

Comment:
Is that accidentally there?

Reply:
it was rebased on the other branch with failing test fix/investigations - rebased back to main